src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -92,16 +93,19 @@ internal void ValidateAccepts(ISymbolSlot slot, FormulaType newType)
         /// <param name="mutable"></param>
         /// <param name="displayName"></param>
         public ISymbolSlot AddVariable(string name, FormulaType type, bool mutable = false, string displayName = null)
-        {
-            if (displayName != null)
-            {
-                // Include parameter so that it's not a breaking change when we enable.
-                // https://github.com/microsoft/Power-Fx/issues/779
-                throw new NotImplementedException("DisplayName support for variables not implemented yet");
-            }
-
+        {
             Inc();
             ValidateName(name);

```

Comment:
>ValidateName(name); [](http://example.com/codeflow?start=12&length=19)

var dName = ValidateName(name);

You can use this are line 107 and 136

Reply:
Great point, I'll move 98 to 103, but will keep 136 as is since its already been validated.